resources/views/emails/exposure.blade.php

```diff
@@ -0,0 +1,13 @@
+@component('mail::message')
+# Hello {{ $first_name }},
+
+You are receiving this message because you visited the Student Competition Center {{ $date }} according to the People Counter system.
+Please see below for an important message from Dr. Cunefare.
+
+@component('mail::panel')
+{{ $body }}
+@endcomponent
+
+Thanks,
+SCC Governing Board
```

Comment:
This is also all on one line when rendered. I would rather leave this off entirely, I don't think it makes sense in context.

Reply:
That's fine, can you add a suggestion to remove it so I can batch commit?